### PR TITLE
made styling changes

### PR DESCRIPTION
**Pull request by diuguide**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,8 +1,22 @@
 .container {
+    background-color: #eddcd2;
```
Comment (thadkingcole) [inline css/style.css:3]:
```suggestion
remove this blank line
```

File: css/style.css [hunk 2]
```diff
@@ -1,8 +1,22 @@
...
     text-align: center;
 
+}
+.title {
+    background-color: #b7b7a4;
+    text-align: center;
+    border-radius: 5px;
+    margin: 3px;
+    margin-top: 5px;
+    color: #362c28;
```
Comment (thadkingcole) [inline css/style.css]: The dark text on the darker background results in the title not standing out as much as the rest of the page. Perhaps making the text color ligher would help. Alternatively, the title background could be made lighter.
```suggestion
    color: white;
```

File: css/style.css [hunk 3]
```diff
@@ -1,8 +1,22 @@
...
+    padding: 5px
 }
```
Comment (thadkingcole) [inline css/style.css]: overall style comments:
I don't really like the white bars on the side of the page. I think they should match the background color in the middle, but they could be a different color if desired. Regardless, they should not be left un-styled as they currently are.

optional formatting suggestions (feel free to ignore):

- add new lines between css classes for better readability.
- add new line at end of file.

**Review by thadkingcole — CHANGES_REQUESTED** [review]
see comments on style.css for some minor changes. I'll be happy to approve after those comments are addressed. let me know if you have any questions.

File: css/style.css (hunk 3 above)
Comment (TheWintersFox) [reply css/style.css]: I agree with both of the optional comments.  

As well I agree with Thads feedback on the white space surrounding the app.  We could probably add in some images around these areas, or choose to extend the app to go to the whole page.  I think adding in images in this space would be the better option here

File: css/style.css (hunk 2 above)
Comment (TheWintersFox) [reply css/style.css]: I don't really prefer the brown background but you were probably adding that in as a placeholder Everett, we can review this later tonight as a group.

File: css/style.css (hunk 3 above)
Comment (diuguide) [reply css/style.css]: white space will be addressed when desktop layout is completed, current layout is based on using iPhone X in dev tools.

**Review by thadkingcole — APPROVED** [review]
Thanks for addressing my earlier comments.

**Pull request merged**